responses/14_congratulations.md

```diff
@@ -14,4 +14,4 @@ You finished this course. You can see a deployed version of our work together at
 
 I won't track any further events in this repository.
 
-Your next steps are in the [Innersource learning path]({{ learningPathURL }}).
\ No newline at end of file
+Your next steps are in the [Innersource learning path]({{ GHE_HOST }}/{{ course.Owner.login }}/paths/innersource:-theory-to-practice).
```

Comment:
You can use `GITHUB_URL` here for a consistent setup on lab.github.com and on GHES!

```suggestion
Your next steps are in the [Innersource learning path]({{ GITHUB_URL }}/{{ course.Owner.login }}/paths/innersource:-theory-to-practice).
```